data/countries/GU.yaml

```diff
@@ -14,17 +16,18 @@ holidays:
       - US
     # additional holidays
     days:
-      03-05:
+      3rd monday in February: false
+      1st monday in March:
         name:
-          en: Guam Discovery Day
+          en: Guam History and Chamorro Heritage Day
       easter -2:
         _name: easter -2
-      07-21:
+        type: observance
+      07-21 if sunday then next monday:
         name:
           en: Liberation Day
       11-02:
         _name: 11-02
-      12-08:
+      12-08 if sunday then next monday:
         name:
-          en: Lady of Camarin Day
-
+          en: Santa Marian Kamalen Day
```

Comment:
I would be tempted to keep this as is. The original name seems to be a bit more official as per the legislation. Maybe add a note that this is also known as Santa Marian Kamalen Day?